spectre-meltdown-checker.sh

```diff
@@ -241,11 +242,9 @@ while [ -n "$1" ]; do
 		opt_batch=1
 		opt_verbose=0
 		shift
-	elif [ "$1" = "--nrpe" ]; then
-		opt_nrpe=1
-		opt_batch=0
-		opt_verbose=0
-		shift
+		case "$1" in
+			text|nrpe) opt_batch_format="$1"; shift;;
```

Comment:
Could you add a `*)` case, with an error message to stderr listing the supported --batch options, and exit with failure code ?
When we add more batch types, unsupported ones on old versions would just fail silently otherwise

Reply:
Done. I had to make it **not** error if the following parameter is another flag or nothing.

```sh
$ sudo ./spectre-meltdown-checker.sh --batch json
./spectre-meltdown-checker.sh: error: unknown batch format 'json'
./spectre-meltdown-checker.sh: error: --batch expects a format from: text, nrpe
```